ttnn/cpp/ttnn/operations/ccl/reduce_scatter/reduce_scatter.cpp

```diff
@@ -14,11 +14,12 @@ ttnn::Tensor ExecuteReduceScatter::invoke(
     ttnn::operations::reduction::ReduceType math_op,
     const uint32_t num_links,
     const std::optional<ttnn::MemoryConfig>& memory_config,
+    ttnn::ccl::Topology topology,
```

Comment:
when `ttnn::ccl::Topology::Line` is used, don't we need something similar to `cluster_axis`define how we want to kick off parallel line gathers on MeshTensor?